src/NeighborhoodsPage/NeighborhoodsPage.js

```diff
@@ -5,22 +5,37 @@ import React from 'react';
 import './NeighborhoodsPage.css';
 import Nav from './../Nav/Nav.js'
 import Header from './../Header/Header.js'
+import Card from '../NeighborhoodCard/NeighborhoodCard.js'
+
 
 class NeighborhoodsPage extends React.Component {
   constructor(props) {
-    super()
+    super(props)
     this.state = {
 
     }
```

Comment:
We really aren't utilizing the state in the component, as well as I dont think the app it self needs to hold on to the areas data since listings really doesn't need it. I think the componentDidMount on the app and that fetch could live here.